### DISCUSSION TITLE
DXCluster in same window as QSO live ?

### DISCUSSION THREAD

**Discussion by GregTroar** [magicbug/Cloudlog#2625] [General]
Hi all,

Is there a way to have the cluster integrated in the QSO page ?
Or no other choice but to have two different pages opened ?
It is a must for me to have the cluster visible from the QSO Live so I can quickly double click on a cluster to jump to the frequency and come back on my run frequency.

Thanks a lot
Greg

**Comment by int2001** [comment]
You can simply click on the "magnifier-glass" next to Callsign at the QSO-Page.
If you've CAT enabled, Cloudlog will fetch the Call/Details of the Spot on your current QRG.

<img width="191" alt="image of magnifier-glass" src="https://github.com/magicbug/Cloudlog/assets/1410708/6cc193e2-2c58-4b17-9b2c-9e61bd80af87">

For "the other way round", you need to have another browser-tab/window opened. Due to the space at the live-qso-page, there's currently no other possibility for that. maybe we implement an option to toggle between "last-contacts" and "spots" at the QSO-Page. But spots are much more/longer than the previous contacts.

HTH

**Reply by GregTroar** [reply]
Yes thanks for the hint on the first part but it is not really my requirement.
It is more when I am running on a CQ frequency so I can keep an eye on the cluster if I see something of interest I can quickly qsy work the DX and come back on my freq so it is the second part of your message.
Indeed it would be very convenient to be able to switch between last contacts and spots even if spots is limited in term of columns available but band freq call country spotter continent or spotter call would be already quite enough to work.

Thanks

**Reply by GregTroar** [reply]
Coming back on this yes a kind of "tab" to switch between last contacts and spots would be very convenient.
As far as the space (width available) is concerned, it can be basic same as last contacts with only 5 columns as long as we know the origin of the spot the call the freq / mode it is enough if we want more details we can always refer to the real cluster page in a separate page.

Anyway this would only be convenient if/when the cat is both ways meaning that we can click on a spot and the radio would switch to the right freq/mode.

Thanks

**Comment by int2001** [comment]
The way Cloudlog to TRX is a bit complicated. @magicbug and i have an Idea for that. In the meantime you can tryout my browser-plugin which syncs the pressed QRG to your FLRig-driven TRX: https://github.com/int2001/dxheat2flrig

**Reply by GregTroar** [reply]
ok I will have a look at your plugin !
Thanks a lot
